core/templates/dev/head/forms/formBuilderSpec.js

```diff
@@ -99,7 +99,14 @@ describe('Normalizer tests', function() {
     expect(filter('-1')).toEqual(-1);
     expect(filter('-1.0')).toEqual(-1);
     expect(filter('1,5')).toEqual(1.5);
-
+    expect(filter('1%')).toEqual(0.01);
+    expect(filter('1.5%')).toEqual(0.015);
+    expect(filter('-5%')).toEqual(-0.05);
+    expect(filter('.35')).toEqual(0.35);
+    expect(filter(',3')).toEqual(0.3);
+
+    expect(filter('3.2% ')).toBeUndefined();
```

Comment:
This is an interesting case and I actually lean towards being permissive and accepting it. Do you think we could do something like 

   var viewValue = input.toString().trim();

in line 911?